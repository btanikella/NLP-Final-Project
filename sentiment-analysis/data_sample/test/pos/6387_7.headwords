As
seems
sequels
classified
movie
movies
