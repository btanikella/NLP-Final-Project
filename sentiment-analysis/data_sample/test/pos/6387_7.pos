IN JJS NNS PRP VBP VBN IN DT IN DT NNS JJ NNS WDT NNP VBZ
EX VBZ TO VB DT NN IN NNP
NNS RB VBP TO VB NN IN NNS VBP NNS WDT VBP DT NN VBN NNP NN VBZ DT JJ NN
PRP MD VB VBN IN JJ NN IN PRP VBZ NN CC VBP RB VB IN VB PRP RP DT NN
DT VBZ DT RB JJ NN WDT VBZ RB VB RBS NNS NNP NNP PRP VBP VBG DT  PRP RB VBP DT JJ NN IN PRP MD RB VB VBG IN DT
DT JJ NNS WDT VBP TO VB TO VB DT TO VBP  NNP CD  CC  NNP IN NNP  WDT VBP JJ JJ NNS WDT VBP DT JJ NN IN JJS IN DT NNP NNP VBZ RB JJ VBG IN PRP VBP VBN IN VBG JJR IN DT NN
