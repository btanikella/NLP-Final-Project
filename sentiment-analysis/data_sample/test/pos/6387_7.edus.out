Sentence #1 (17 tokens):
As most people I am tired of the by the numbers clichd movies that Hollywood makes.
[Text=As CharacterOffsetBegin=4 CharacterOffsetEnd=6 PartOfSpeech=IN] [Text=most CharacterOffsetBegin=7 CharacterOffsetEnd=11 PartOfSpeech=JJS] [Text=people CharacterOffsetBegin=12 CharacterOffsetEnd=18 PartOfSpeech=NNS] [Text=I CharacterOffsetBegin=19 CharacterOffsetEnd=20 PartOfSpeech=PRP] [Text=am CharacterOffsetBegin=21 CharacterOffsetEnd=23 PartOfSpeech=VBP] [Text=tired CharacterOffsetBegin=24 CharacterOffsetEnd=29 PartOfSpeech=VBN] [Text=of CharacterOffsetBegin=30 CharacterOffsetEnd=32 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=33 CharacterOffsetEnd=36 PartOfSpeech=DT] [Text=by CharacterOffsetBegin=37 CharacterOffsetEnd=39 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=40 CharacterOffsetEnd=43 PartOfSpeech=DT] [Text=numbers CharacterOffsetBegin=44 CharacterOffsetEnd=51 PartOfSpeech=NNS] [Text=clichd CharacterOffsetBegin=52 CharacterOffsetEnd=58 PartOfSpeech=JJ] [Text=movies CharacterOffsetBegin=59 CharacterOffsetEnd=65 PartOfSpeech=NNS] [Text=that CharacterOffsetBegin=66 CharacterOffsetEnd=70 PartOfSpeech=WDT] [Text=Hollywood CharacterOffsetBegin=71 CharacterOffsetEnd=80 PartOfSpeech=NNP] [Text=makes CharacterOffsetBegin=81 CharacterOffsetEnd=86 PartOfSpeech=VBZ] [Text=. CharacterOffsetBegin=86 CharacterOffsetEnd=87 PartOfSpeech=.] 
root(ROOT-0, As-1)
amod(people-3, most-2)
pobj(As-1, people-3)
nsubjpass(tired-6, I-4)
auxpass(tired-6, am-5)
rcmod(people-3, tired-6)
prep(tired-6, of-7)
pobj(of-7, the-8)
prep(the-8, by-9)
det(numbers-11, the-10)
pobj(by-9, numbers-11)
amod(movies-13, clichd-12)
dep(numbers-11, movies-13)
dobj(makes-16, that-14)
nsubj(makes-16, Hollywood-15)
rcmod(movies-13, makes-16)
punct(As-1, .-17)

Sentence #2 (9 tokens):
There seems to be no creativity in Hollywood.
[Text=There CharacterOffsetBegin=92 CharacterOffsetEnd=97 PartOfSpeech=EX] [Text=seems CharacterOffsetBegin=98 CharacterOffsetEnd=103 PartOfSpeech=VBZ] [Text=to CharacterOffsetBegin=104 CharacterOffsetEnd=106 PartOfSpeech=TO] [Text=be CharacterOffsetBegin=107 CharacterOffsetEnd=109 PartOfSpeech=VB] [Text=no CharacterOffsetBegin=110 CharacterOffsetEnd=112 PartOfSpeech=DT] [Text=creativity CharacterOffsetBegin=113 CharacterOffsetEnd=123 PartOfSpeech=NN] [Text=in CharacterOffsetBegin=124 CharacterOffsetEnd=126 PartOfSpeech=IN] [Text=Hollywood CharacterOffsetBegin=127 CharacterOffsetEnd=136 PartOfSpeech=NNP] [Text=. CharacterOffsetBegin=136 CharacterOffsetEnd=137 PartOfSpeech=.] 
root(ROOT-0, seems-2)
expl(seems-2, There-1)
aux(creativity-6, to-3)
cop(creativity-6, be-4)
det(creativity-6, no-5)
xcomp(seems-2, creativity-6)
prep(creativity-6, in-7)
pobj(in-7, Hollywood-8)
punct(seems-2, .-9)

Sentence #3 (22 tokens):
Companies only want to spend money on remakes are sequels that have an audience built in.This movie is a welcome change.
[Text=Companies CharacterOffsetBegin=142 CharacterOffsetEnd=151 PartOfSpeech=NNS] [Text=only CharacterOffsetBegin=152 CharacterOffsetEnd=156 PartOfSpeech=RB] [Text=want CharacterOffsetBegin=157 CharacterOffsetEnd=161 PartOfSpeech=VBP] [Text=to CharacterOffsetBegin=162 CharacterOffsetEnd=164 PartOfSpeech=TO] [Text=spend CharacterOffsetBegin=165 CharacterOffsetEnd=170 PartOfSpeech=VB] [Text=money CharacterOffsetBegin=171 CharacterOffsetEnd=176 PartOfSpeech=NN] [Text=on CharacterOffsetBegin=177 CharacterOffsetEnd=179 PartOfSpeech=IN] [Text=remakes CharacterOffsetBegin=180 CharacterOffsetEnd=187 PartOfSpeech=NNS] [Text=are CharacterOffsetBegin=188 CharacterOffsetEnd=191 PartOfSpeech=VBP] [Text=sequels CharacterOffsetBegin=192 CharacterOffsetEnd=199 PartOfSpeech=NNS] [Text=that CharacterOffsetBegin=200 CharacterOffsetEnd=204 PartOfSpeech=WDT] [Text=have CharacterOffsetBegin=205 CharacterOffsetEnd=209 PartOfSpeech=VBP] [Text=an CharacterOffsetBegin=210 CharacterOffsetEnd=212 PartOfSpeech=DT] [Text=audience CharacterOffsetBegin=213 CharacterOffsetEnd=221 PartOfSpeech=NN] [Text=built CharacterOffsetBegin=222 CharacterOffsetEnd=227 PartOfSpeech=VBN] [Text=in.This CharacterOffsetBegin=228 CharacterOffsetEnd=235 PartOfSpeech=NNP] [Text=movie CharacterOffsetBegin=236 CharacterOffsetEnd=241 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=242 CharacterOffsetEnd=244 PartOfSpeech=VBZ] [Text=a CharacterOffsetBegin=245 CharacterOffsetEnd=246 PartOfSpeech=DT] [Text=welcome CharacterOffsetBegin=247 CharacterOffsetEnd=254 PartOfSpeech=JJ] [Text=change CharacterOffsetBegin=255 CharacterOffsetEnd=261 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=261 CharacterOffsetEnd=262 PartOfSpeech=.] 
root(ROOT-0, sequels-10)
nsubj(want-3, Companies-1)
advmod(want-3, only-2)
csubj(sequels-10, want-3)
aux(spend-5, to-4)
xcomp(want-3, spend-5)
dobj(spend-5, money-6)
prep(spend-5, on-7)
pobj(on-7, remakes-8)
cop(sequels-10, are-9)
nsubj(have-12, that-11)
rcmod(sequels-10, have-12)
det(audience-14, an-13)
dobj(have-12, audience-14)
amod(movie-17, built-15)
nn(movie-17, in.This-16)
nsubj(change-21, movie-17)
cop(change-21, is-18)
det(change-21, a-19)
amod(change-21, welcome-20)
rcmod(audience-14, change-21)
punct(sequels-10, .-22)

Sentence #4 (22 tokens):
It could be classified as romantic comedy for it's genre but don't let that turn you off this movie.
[Text=It CharacterOffsetBegin=267 CharacterOffsetEnd=269 PartOfSpeech=PRP] [Text=could CharacterOffsetBegin=270 CharacterOffsetEnd=275 PartOfSpeech=MD] [Text=be CharacterOffsetBegin=276 CharacterOffsetEnd=278 PartOfSpeech=VB] [Text=classified CharacterOffsetBegin=279 CharacterOffsetEnd=289 PartOfSpeech=VBN] [Text=as CharacterOffsetBegin=290 CharacterOffsetEnd=292 PartOfSpeech=IN] [Text=romantic CharacterOffsetBegin=293 CharacterOffsetEnd=301 PartOfSpeech=JJ] [Text=comedy CharacterOffsetBegin=302 CharacterOffsetEnd=308 PartOfSpeech=NN] [Text=for CharacterOffsetBegin=309 CharacterOffsetEnd=312 PartOfSpeech=IN] [Text=it CharacterOffsetBegin=313 CharacterOffsetEnd=315 PartOfSpeech=PRP] [Text='s CharacterOffsetBegin=315 CharacterOffsetEnd=317 PartOfSpeech=VBZ] [Text=genre CharacterOffsetBegin=318 CharacterOffsetEnd=323 PartOfSpeech=NN] [Text=but CharacterOffsetBegin=324 CharacterOffsetEnd=327 PartOfSpeech=CC] [Text=do CharacterOffsetBegin=328 CharacterOffsetEnd=330 PartOfSpeech=VBP] [Text=n't CharacterOffsetBegin=330 CharacterOffsetEnd=333 PartOfSpeech=RB] [Text=let CharacterOffsetBegin=334 CharacterOffsetEnd=337 PartOfSpeech=VB] [Text=that CharacterOffsetBegin=338 CharacterOffsetEnd=342 PartOfSpeech=IN] [Text=turn CharacterOffsetBegin=343 CharacterOffsetEnd=347 PartOfSpeech=VB] [Text=you CharacterOffsetBegin=348 CharacterOffsetEnd=351 PartOfSpeech=PRP] [Text=off CharacterOffsetBegin=352 CharacterOffsetEnd=355 PartOfSpeech=RP] [Text=this CharacterOffsetBegin=356 CharacterOffsetEnd=360 PartOfSpeech=DT] [Text=movie CharacterOffsetBegin=361 CharacterOffsetEnd=366 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=366 CharacterOffsetEnd=367 PartOfSpeech=.] 
root(ROOT-0, classified-4)
nsubjpass(classified-4, It-1)
aux(classified-4, could-2)
auxpass(classified-4, be-3)
prep(classified-4, as-5)
amod(comedy-7, romantic-6)
pobj(as-5, comedy-7)
mark(genre-11, for-8)
nsubj(genre-11, it-9)
cop(genre-11, 's-10)
rcmod(comedy-7, genre-11)
cc(genre-11, but-12)
aux(let-15, do-13)
neg(let-15, n't-14)
conj(genre-11, let-15)
nsubj(turn-17, that-16)
dep(genre-11, turn-17)
dobj(turn-17, you-18)
prt(turn-17, off-19)
det(movie-21, this-20)
dobj(turn-17, movie-21)
punct(classified-4, .-22)

Sentence #5 (34 tokens):
This is a very original movie which is not like most things Hollywood produces.If you are reading this, you already know the basic plot so I will not bother going over that.
[Text=This CharacterOffsetBegin=372 CharacterOffsetEnd=376 PartOfSpeech=DT] [Text=is CharacterOffsetBegin=377 CharacterOffsetEnd=379 PartOfSpeech=VBZ] [Text=a CharacterOffsetBegin=380 CharacterOffsetEnd=381 PartOfSpeech=DT] [Text=very CharacterOffsetBegin=382 CharacterOffsetEnd=386 PartOfSpeech=RB] [Text=original CharacterOffsetBegin=387 CharacterOffsetEnd=395 PartOfSpeech=JJ] [Text=movie CharacterOffsetBegin=396 CharacterOffsetEnd=401 PartOfSpeech=NN] [Text=which CharacterOffsetBegin=402 CharacterOffsetEnd=407 PartOfSpeech=WDT] [Text=is CharacterOffsetBegin=408 CharacterOffsetEnd=410 PartOfSpeech=VBZ] [Text=not CharacterOffsetBegin=411 CharacterOffsetEnd=414 PartOfSpeech=RB] [Text=like CharacterOffsetBegin=415 CharacterOffsetEnd=419 PartOfSpeech=VB] [Text=most CharacterOffsetBegin=420 CharacterOffsetEnd=424 PartOfSpeech=RBS] [Text=things CharacterOffsetBegin=425 CharacterOffsetEnd=431 PartOfSpeech=NNS] [Text=Hollywood CharacterOffsetBegin=432 CharacterOffsetEnd=441 PartOfSpeech=NNP] [Text=produces.If CharacterOffsetBegin=442 CharacterOffsetEnd=453 PartOfSpeech=NNP] [Text=you CharacterOffsetBegin=454 CharacterOffsetEnd=457 PartOfSpeech=PRP] [Text=are CharacterOffsetBegin=458 CharacterOffsetEnd=461 PartOfSpeech=VBP] [Text=reading CharacterOffsetBegin=462 CharacterOffsetEnd=469 PartOfSpeech=VBG] [Text=this CharacterOffsetBegin=470 CharacterOffsetEnd=474 PartOfSpeech=DT] [Text=, CharacterOffsetBegin=474 CharacterOffsetEnd=475 PartOfSpeech=,] [Text=you CharacterOffsetBegin=476 CharacterOffsetEnd=479 PartOfSpeech=PRP] [Text=already CharacterOffsetBegin=480 CharacterOffsetEnd=487 PartOfSpeech=RB] [Text=know CharacterOffsetBegin=488 CharacterOffsetEnd=492 PartOfSpeech=VBP] [Text=the CharacterOffsetBegin=493 CharacterOffsetEnd=496 PartOfSpeech=DT] [Text=basic CharacterOffsetBegin=497 CharacterOffsetEnd=502 PartOfSpeech=JJ] [Text=plot CharacterOffsetBegin=503 CharacterOffsetEnd=507 PartOfSpeech=NN] [Text=so CharacterOffsetBegin=508 CharacterOffsetEnd=510 PartOfSpeech=IN] [Text=I CharacterOffsetBegin=511 CharacterOffsetEnd=512 PartOfSpeech=PRP] [Text=will CharacterOffsetBegin=513 CharacterOffsetEnd=517 PartOfSpeech=MD] [Text=not CharacterOffsetBegin=518 CharacterOffsetEnd=521 PartOfSpeech=RB] [Text=bother CharacterOffsetBegin=522 CharacterOffsetEnd=528 PartOfSpeech=VB] [Text=going CharacterOffsetBegin=529 CharacterOffsetEnd=534 PartOfSpeech=VBG] [Text=over CharacterOffsetBegin=535 CharacterOffsetEnd=539 PartOfSpeech=IN] [Text=that CharacterOffsetBegin=540 CharacterOffsetEnd=544 PartOfSpeech=DT] [Text=. CharacterOffsetBegin=544 CharacterOffsetEnd=545 PartOfSpeech=.] 
root(ROOT-0, movie-6)
nsubj(movie-6, This-1)
cop(movie-6, is-2)
det(movie-6, a-3)
advmod(original-5, very-4)
amod(movie-6, original-5)
nsubj(like-10, which-7)
aux(like-10, is-8)
neg(like-10, not-9)
rcmod(movie-6, like-10)
amod(things-12, most-11)
dobj(like-10, things-12)
nn(produces.If-14, Hollywood-13)
dep(things-12, produces.If-14)
nsubj(reading-17, you-15)
aux(reading-17, are-16)
rcmod(produces.If-14, reading-17)
dobj(reading-17, this-18)
punct(movie-6, ,-19)
nsubj(know-22, you-20)
advmod(know-22, already-21)
rcmod(movie-6, know-22)
det(plot-25, the-23)
amod(plot-25, basic-24)
dobj(know-22, plot-25)
mark(bother-30, so-26)
nsubj(bother-30, I-27)
aux(bother-30, will-28)
neg(bother-30, not-29)
advcl(know-22, bother-30)
xcomp(bother-30, going-31)
prep(going-31, over-32)
pobj(over-32, that-33)
punct(movie-6, .-34)

Sentence #6 (52 tokens):
The only movies that come to mind to compare this to are "Interstate 60" and "Art of Travel" which are little known gems that take a different path than most of the Hollywood garbage.This is well worth seeing if you are tired of watching more of the same.Dean
[Text=The CharacterOffsetBegin=550 CharacterOffsetEnd=553 PartOfSpeech=DT] [Text=only CharacterOffsetBegin=554 CharacterOffsetEnd=558 PartOfSpeech=JJ] [Text=movies CharacterOffsetBegin=559 CharacterOffsetEnd=565 PartOfSpeech=NNS] [Text=that CharacterOffsetBegin=566 CharacterOffsetEnd=570 PartOfSpeech=WDT] [Text=come CharacterOffsetBegin=571 CharacterOffsetEnd=575 PartOfSpeech=VBP] [Text=to CharacterOffsetBegin=576 CharacterOffsetEnd=578 PartOfSpeech=TO] [Text=mind CharacterOffsetBegin=579 CharacterOffsetEnd=583 PartOfSpeech=VB] [Text=to CharacterOffsetBegin=584 CharacterOffsetEnd=586 PartOfSpeech=TO] [Text=compare CharacterOffsetBegin=587 CharacterOffsetEnd=594 PartOfSpeech=VB] [Text=this CharacterOffsetBegin=595 CharacterOffsetEnd=599 PartOfSpeech=DT] [Text=to CharacterOffsetBegin=600 CharacterOffsetEnd=602 PartOfSpeech=TO] [Text=are CharacterOffsetBegin=603 CharacterOffsetEnd=606 PartOfSpeech=VBP] [Text=`` CharacterOffsetBegin=607 CharacterOffsetEnd=608 PartOfSpeech=``] [Text=Interstate CharacterOffsetBegin=608 CharacterOffsetEnd=618 PartOfSpeech=NNP] [Text=60 CharacterOffsetBegin=619 CharacterOffsetEnd=621 PartOfSpeech=CD] [Text='' CharacterOffsetBegin=621 CharacterOffsetEnd=622 PartOfSpeech=''] [Text=and CharacterOffsetBegin=623 CharacterOffsetEnd=626 PartOfSpeech=CC] [Text=`` CharacterOffsetBegin=627 CharacterOffsetEnd=628 PartOfSpeech=``] [Text=Art CharacterOffsetBegin=628 CharacterOffsetEnd=631 PartOfSpeech=NNP] [Text=of CharacterOffsetBegin=632 CharacterOffsetEnd=634 PartOfSpeech=IN] [Text=Travel CharacterOffsetBegin=635 CharacterOffsetEnd=641 PartOfSpeech=NNP] [Text='' CharacterOffsetBegin=641 CharacterOffsetEnd=642 PartOfSpeech=''] [Text=which CharacterOffsetBegin=643 CharacterOffsetEnd=648 PartOfSpeech=WDT] [Text=are CharacterOffsetBegin=649 CharacterOffsetEnd=652 PartOfSpeech=VBP] [Text=little CharacterOffsetBegin=653 CharacterOffsetEnd=659 PartOfSpeech=JJ] [Text=known CharacterOffsetBegin=660 CharacterOffsetEnd=665 PartOfSpeech=JJ] [Text=gems CharacterOffsetBegin=666 CharacterOffsetEnd=670 PartOfSpeech=NNS] [Text=that CharacterOffsetBegin=671 CharacterOffsetEnd=675 PartOfSpeech=WDT] [Text=take CharacterOffsetBegin=676 CharacterOffsetEnd=680 PartOfSpeech=VBP] [Text=a CharacterOffsetBegin=681 CharacterOffsetEnd=682 PartOfSpeech=DT] [Text=different CharacterOffsetBegin=683 CharacterOffsetEnd=692 PartOfSpeech=JJ] [Text=path CharacterOffsetBegin=693 CharacterOffsetEnd=697 PartOfSpeech=NN] [Text=than CharacterOffsetBegin=698 CharacterOffsetEnd=702 PartOfSpeech=IN] [Text=most CharacterOffsetBegin=703 CharacterOffsetEnd=707 PartOfSpeech=JJS] [Text=of CharacterOffsetBegin=708 CharacterOffsetEnd=710 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=711 CharacterOffsetEnd=714 PartOfSpeech=DT] [Text=Hollywood CharacterOffsetBegin=715 CharacterOffsetEnd=724 PartOfSpeech=NNP] [Text=garbage.This CharacterOffsetBegin=725 CharacterOffsetEnd=737 PartOfSpeech=NNP] [Text=is CharacterOffsetBegin=738 CharacterOffsetEnd=740 PartOfSpeech=VBZ] [Text=well CharacterOffsetBegin=741 CharacterOffsetEnd=745 PartOfSpeech=RB] [Text=worth CharacterOffsetBegin=746 CharacterOffsetEnd=751 PartOfSpeech=JJ] [Text=seeing CharacterOffsetBegin=752 CharacterOffsetEnd=758 PartOfSpeech=VBG] [Text=if CharacterOffsetBegin=759 CharacterOffsetEnd=761 PartOfSpeech=IN] [Text=you CharacterOffsetBegin=762 CharacterOffsetEnd=765 PartOfSpeech=PRP] [Text=are CharacterOffsetBegin=766 CharacterOffsetEnd=769 PartOfSpeech=VBP] [Text=tired CharacterOffsetBegin=770 CharacterOffsetEnd=775 PartOfSpeech=VBN] [Text=of CharacterOffsetBegin=776 CharacterOffsetEnd=778 PartOfSpeech=IN] [Text=watching CharacterOffsetBegin=779 CharacterOffsetEnd=787 PartOfSpeech=VBG] [Text=more CharacterOffsetBegin=788 CharacterOffsetEnd=792 PartOfSpeech=JJR] [Text=of CharacterOffsetBegin=793 CharacterOffsetEnd=795 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=796 CharacterOffsetEnd=799 PartOfSpeech=DT] [Text=same.Dean CharacterOffsetBegin=800 CharacterOffsetEnd=809 PartOfSpeech=NN] 
root(ROOT-0, movies-3)
det(movies-3, The-1)
amod(movies-3, only-2)
nsubj(come-5, that-4)
rcmod(movies-3, come-5)
prep(come-5, to-6)
pobj(to-6, mind-7)
aux(compare-9, to-8)
xcomp(come-5, compare-9)
dobj(compare-9, this-10)
aux(Interstate-14, to-11)
cop(Interstate-14, are-12)
punct(Interstate-14, ``-13)
xcomp(compare-9, Interstate-14)
num(Interstate-14, 60-15)
punct(Interstate-14, ''-16)
cc(Interstate-14, and-17)
punct(Interstate-14, ``-18)
conj(Interstate-14, Art-19)
prep(Art-19, of-20)
pobj(of-20, Travel-21)
punct(Art-19, ''-22)
nsubj(little-25, which-23)
cop(little-25, are-24)
rcmod(Art-19, little-25)
amod(gems-27, known-26)
dep(little-25, gems-27)
nsubj(take-29, that-28)
rcmod(gems-27, take-29)
det(path-32, a-30)
amod(path-32, different-31)
dobj(take-29, path-32)
mark(worth-41, than-33)
nsubj(worth-41, most-34)
prep(most-34, of-35)
det(garbage.This-38, the-36)
nn(garbage.This-38, Hollywood-37)
pobj(of-35, garbage.This-38)
cop(worth-41, is-39)
advmod(worth-41, well-40)
dep(path-32, worth-41)
dep(worth-41, seeing-42)
mark(tired-46, if-43)
nsubj(tired-46, you-44)
cop(tired-46, are-45)
advcl(worth-41, tired-46)
prep(tired-46, of-47)
pcomp(of-47, watching-48)
dobj(watching-48, more-49)
prep(more-49, of-50)
det(same.Dean-52, the-51)
pobj(of-50, same.Dean-52)

